types/slate-react/index.d.ts

```diff
@@ -128,16 +136,287 @@ export class Editor extends React.Component<EditorProps, EditorState> {
     readonly readOnly: boolean;
     readonly value: Value;
 
-    // Instance Methods
-    applyOperation(...args: any[]): Controller;
-    blur(): void;
-    command(...args: any[]): Controller;
-    focus(): void;
-    normalize(...args: any[]): Controller;
-    query(...args: any[]): Controller;
+    // Instance methods
     resolveController(plugins: Plugin[], schema: Schema, commands: any[], queries: any[]): void;
-    run(...args: any[]): any;
-    withoutNormalizing(...args: any[]): Controller;
+
+    // Controller
+    addMark(mark: Mark | MarkProperties | string): Editor;
```

Comment:
Thoughts on declaring all of these methods using the existing types on `EditorController`? e.g.

```ts
addMark: Controller['addMark'];
delete: Controller['delete'];
// etc
```

I think this will help to keep the two classes in sync as command signatures change over time, with the `EditorController` as the source of truth.

Reply:
I quite like this idea, I'll give it a shot

Reply:
I've updated the PR to use this syntax, thanks for the suggestion!